ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -652,7 +652,11 @@ export class NodeDetailsContent extends React.Component {
 
   availableFields = () => {
     if (this.props.dynamicParameterDefinitions) {
-      return this.props.dynamicParameterDefinitions.map(param => param.name)
+      const commonFields = ["id", "outputVar"]
```

Comment:
can we extract common method with Join handling below?

Reply:
good point, extracted